.github/CHANGELOG.md

```diff
@@ -2,6 +2,55 @@
 
 <h3>New features since last release</h3>
 
+* The transform for the Jacobian of the classical preprocessing within a QNode,
+  `qml.transforms.classical_jacobian`, now takes a keyword argument `argnums` to specify
+  the QNode argument indices w.r.t which the Jacobian is computed.
+  [(#1645)](https://github.com/PennyLaneAI/pennylane/pull/1645)
+
+  The output behaviour with `argnums=None` differs slightly for different interfaces,
+  following the respective output shape of the native `jacobian` equivalent:
+
+  | Interface    | `argnums=None`     | `type(argnums)=int`   | `argnums=Sequence[int]`   |
+  | ------------ | ------------------ | --------------------- | ------------------------- |
+  | `'autograd'` | `tuple(arrays)`\*  | `array`               | `tuple(array)`            |
+  | `'jax'`      | `array`            | `array`               | `tuple(array)`            |
+  | `'tf'`       | `tuple(arrays)`    | `array`               | `tuple(array)`            |
+  | `'torch'`    | `tuple(arrays)`    | `array`               | `tuple(array)`            |
```

Comment:
The table doesn't seem to render, perhaps due to some issue with our markdown -> ReST conversion?

Reply:
![image](https://user-images.githubusercontent.com/2959003/133027316-f40fefb3-5124-45e1-ad8d-edeee37038ab.png)

Reply:
(The table is a really nice idea though!)